ccd-definition/ComplexTypes/CareSupervision/JudicialMessage/JudicialMessage.json

```diff
@@ -133,15 +133,15 @@
     "ListElementCode": "messageHistory",
     "FieldType": "TextArea",
     "ElementLabel": "Message history",
-    "SecurityClassification": "Public",
-    "FieldShowCondition": "status=\"CLOSED\""
+    "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "JudicialMessage",
     "ListElementCode": "closeMessageLabel",
     "FieldType": "Label",
     "ElementLabel": "This message will now be marked as closed",
-    "SecurityClassification": "Public"
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "closeMessageLabel=\"DO_NOT_SHOW\""
```

Comment:
Is this still displayed in the event? I am only guessing but I am assuming this is what it was originally for.

Reply:
I've checked and it still appears in the event as expected due it being in **judicialMessageReply.json**